ext/File-Find/t/find.t

```diff
@@ -1111,5 +1115,5 @@ if ($^O eq 'MSWin32') {
     like($@, qr/invalid top directory/,
         "find() correctly died due to undefined top directory");
 }
-
+chdir $root_dir; # let File::Temp cleanup.
```

Comment:
BTW if there's no back-compat concerns in this file, you could use a `defer` block instead to ensure this always happens even after a failure.

```perl
my $orig_dir = cwd();
defer { chdir $orig_dir; }
```

at the top.

Reply:
Enabling defer seems to require me dealing with a bunch of other stuff id rather not look into just now. So I added a comment. Ill revisit it another time.